pkg/ebpf/write_capture.go

```diff
@@ -19,15 +19,16 @@ func (t *Tracee) processFileWrites(ctx context.Context) {
 	logger.Debugw("Starting processFileWrites go routine")
 	defer logger.Debugw("Stopped processFileWrites go routine")
 
+	// revive-ignore
 	const (
 		// S_IFMT uint32 = 0170000 // bit mask for the file type bit field
-		S_IFSOCK uint32 = 0140000 // socket
-		S_IFLNK  uint32 = 0120000 // symbolic link
-		S_IFREG  uint32 = 0100000 // regular file
-		S_IFBLK  uint32 = 0060000 // block device
-		S_IFDIR  uint32 = 0040000 // directory
-		S_IFCHR  uint32 = 0020000 // character device
-		S_IFIFO  uint32 = 0010000 // FIFO
+		s_IFSOCK uint32 = 0140000 // socket
+		s_IFLNK  uint32 = 0120000 // symbolic link
+		s_IFREG  uint32 = 0100000 // regular file
+		s_IFBLK  uint32 = 0060000 // block device
+		s_IFDIR  uint32 = 0040000 // directory
+		s_IFCHR  uint32 = 0020000 // character device
+		s_IFIFO  uint32 = 0010000 // FIFO
```

Comment:
I guess the up ignore didn't work here. Constants from C would be converted to Golang formatting like: https://github.com/aquasecurity/tracee/blob/2de14ea130db3e76b031218e273187978a4b6eb1/pkg/ebpf/bpf_log.go#L18-L31

WDYT?

Reply:
hum... yep, I wasn't sure about the constant naming and the linting. I didn't want to "ignore" because I think it will end up making us to have too many ignores around the code. Maybe to add a known prefix (upper/lower case depending on whether its exported or not).